fdbserver/workloads/WriteTagThrottling.actor.cpp

```diff
@@ -123,9 +123,8 @@ struct WriteTagThrottlingWorkload : KVWorkload {
 			return false;
 		}
 		if (writeThrottle) {
-			if (!badActorThrottleRetries) {
-				TraceEvent(SevError, "NoThrottleTriggered");
-				return false;
+			if (!badActorThrottleRetries && !goodActorThrottleRetries) {
+				TraceEvent(SevWarn, "NoThrottleTriggered");
```

Comment:
interesting. Before the change, did you see any simulation failure?

Reply:
Yes. In fact, I think whether throttling is triggered is depended on the composition of testing parameters... If I decrease trPerSecond, or increase tpsRate, chances are the pressure is not enough to lead to throttling. Here I just can guarantee that if throttling happens, badActors have to be limited first.